src/main/java/com/google/devtools/build/docgen/templates/attributes/common/target_compatible_with.html

```diff
@@ -0,0 +1,75 @@
+<p>
```

Comment:
I added this so it shows up on https://docs.bazel.build/versions/master/be/common-definitions.html.

Should I move the examples to https://docs.bazel.build/versions/master/platforms.html? It seemed wrong to me if we're aiming to expand this feature to also include things like "is GCC".